packages/create-redwood-app/package.json

```diff
@@ -14,8 +15,7 @@
   ],
   "scripts": {
     "build": "yarn node ./build.mjs",
-    "build:watch": "nodemon --watch src --ignore dist,template --exec \"yarn build\"",
-    "prepublishOnly": "NODE_ENV=production yarn build"
+    "test": "yarn node ./tests/template.test.js"
```

Comment:
I tried `yarn node --test` at first, but when we run `yarn test` from the root, this command fails because it's fed bad options (jest related ones, like colors and maxWorkers). Something to figure out, but for now this works